security/authentication.md

```diff
@@ -4,13 +4,17 @@ description: How to use JSON Web Tokens
 
 # Authentication
 
+The API supports the "Basic" HTTP authentication scheme [RFC 7617](https://tools.ietf.org/html/rfc7617) with [JSON Web Tokens](https://jwt.io/) [RFC 7519](https://tools.ietf.org/html/rfc7519)
```

Comment:
We're not actually supporting JWT tokens yet, right?

Reply:
We are using them as authorization token after successful basic authentication.
Improved text